native/cocos/bindings/manual/jsb_global_init.cpp

```diff
@@ -197,6 +197,8 @@ void jsb_init_file_operation_delegate() { //NOLINT
         CC_ASSERT(delegate.isValid());
 
         se::ScriptEngine::getInstance()->setFileOperationDelegate(delegate);
+    } else {
+        se::ScriptEngine::getInstance()->setFileOperationDelegate(delegate); // game maybe relaunched at different thread but at the same process
```

Comment:
When does this happen?

Reply:
While game launch from recent task, but gameActivity has been destroy by user or system

Reply:
Got it, please add it in comment.